app/lib/account/models.dart

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2019, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:gcloud/db.dart' as db;
+
+/// User data model.
+@db.Kind(name: 'User', idType: db.IdType.String)
+class User extends db.ExpandoModel {
+  String get userId => id as String;
+
+  @db.StringProperty()
+  String authProvider;
+
+  @db.StringProperty()
+  String authUserId;
+
+  @db.StringProperty()
+  String email;
+
+  @db.StringListProperty()
+  List<String> historicalEmails = <String>[];
```

Comment:
I suggest we drop this... What need to do we have? Also what are the privacy implications.

IMO, the only reason we store `email` is because:
 1. We want to be able to contact users
 2. We need a display name that has some trust

Reply:
removed